cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,281 @@ Cask Data Application Platform Release Notes
+
+- `CDAP-4240<https://issues.cask.co/browse/CDAP-4240>`__ -
+  Fixed a bug in the CLI command to load an artifact, where the wrong artifact name and version is
+  used if the artifact name ends with a number.
+
+- `CDAP-4532<https://issues.cask.co/browse/CDAP-4532>`__ -
+  Fixed checkstyle errors on windows
+
+- `CDAP-4294<https://issues.cask.co/browse/CDAP-4294>`__ -
+  Fixed a bug where plugins from another namespace were visible when creating an application using a system artifact.
+
+- `CDAP-4092<https://issues.cask.co/browse/CDAP-4092>`__ -
+  Fixes an issue that causes worker threads to go into an infinite recursion while exceptions
+  are being thrown in channel handlers.
+
+- `CDAP-3924<https://issues.cask.co/browse/CDAP-3924>`__ -
+  Fix QueryClient to work against HTTPS.
+
+- `CDAP-3742<https://issues.cask.co/browse/CDAP-3742>`__ -
+  Fix a bug where certain MapReduce metrics were not being properly emitted when using multiple outputs.
+
+- `CDAP-4067<https://issues.cask.co/browse/CDAP-4067>`__ -
+  Fixed an issue where socket connections to the TransactionManager were not being closed.
+
+- `CDAP-3887<https://issues.cask.co/browse/CDAP-3887>`__ -
+  CDAP Router times out idle connections
+
+- `CDAP-3902<https://issues.cask.co/browse/CDAP-3902>`__ -
+  Fixed a problem with dataset performance degradation due to the making of multiple remote calls
+  for each "get dataset" request
+
+- `CDAP-4366<https://issues.cask.co/browse/CDAP-4366>`__ -
+  Improved error message when stream views are not found.
+
+- `CDAP-4316<https://issues.cask.co/browse/CDAP-4316>`__ -
+  Don't connect to CDAP successfully in CLI when the hostname and port are incorrect.
+
+- `CDAP-3807<https://issues.cask.co/browse/CDAP-3807>`__ -
+  Added fix for case sensitivity handling in DBSink
+
+- `CDAP-3761<https://issues.cask.co/browse/CDAP-3761>`__ -
+  DBSink column names are used to filter input records fields before writing them to DBSink.
+
+- `CDAP-4534<https://issues.cask.co/browse/CDAP-4534>`__ -
+  Added a configuration to exempt certain URLs from authentication.
+
+- `CDAP-2481<https://issues.cask.co/browse/CDAP-2481>`__ -
+  Removed deprecated endpoint to get status of currently running node in a workflow.
+
+- `CDAP-2470<https://issues.cask.co/browse/CDAP-2470>`__ -
+  Added a check to cause a DB (source or sink) pipeline to fail during deployment if the table (source or sink)
+  was not found, or if an incorrect connection string was provided.
+
+- `CDAP-3815<https://issues.cask.co/browse/CDAP-3815>`__ -
+  Fixed an issue where the regex filter for S3 Batch Source wasn't getting applied correctly.
+
+- `CDAP-4000<https://issues.cask.co/browse/CDAP-4000>`__ -
+  Fixed an issue where a stream that has a view could not be deleted cleanly.
+
+- `CDAP-1174<https://issues.cask.co/browse/CDAP-1174>`__ -
+  Prefix start script functions with cdap to prevent namespace collisions
```

Comment:
init script change. chris - "I've already hit an issue where CDAP's "set_hive_classpath" function conflicted with another "set_hive_classpath" in the framework I was launching it from."